server/tools/scenery.js

```diff
@@ -87,14 +87,54 @@ function deleteModel(workspace, body) {
     return true;
 }
 
-function createDefinition(workspace, body) {
-    console.log("Create Definition: " + workspace + ": " + json(body));
-    return false;
+function createDefinition(workspace, body) { 
```

Comment:
Can you rebase the PR so that it only shows the latest changes?

Reply:
sure thing